### ISSUE TITLE
Increase the number of vehicle flow in generator 

### ISSUE THREAD

**Issue by oroojlooy** [cityflow-project/CityFlow#77]
When I run the generator, it by default only generates four vehicle data in the `flow.json`. How can I increase this?

**Comment by zhc134** [comment]
You can change `--interval` (seconds between two consecutive vehicles) to increase the number of vehicles.
By default, the generator will generate vehicles with straight routes (without any turns). If you turn on `--turn`, it will create 8 more routes (from one corner to opposite corner).
The generator is a tool for simple roadnet and flow. If you need more complicated routes, you need to add them to `flow.json` by yourself.

**Comment by oroojlooy** [comment]
Thanks for the reply. I added `--interval=0.3` (it was `2` by default) and added `--turn` too. But, still it only includes **four** vehicle records. Any suggestion?

**Comment by zhc134** [comment]
When I run `python generate_grid_scenario.py 1 1 --turn`, there are 12 flows in `flow.json`.
What do you mean by **four** vehicle records?
Also to notice, if you set interval to 0.3, then when running simulation, the `interval` in config file should be smaller values like 0.1 or 0.3, it won't generate as much vehicles as you want if you set the simulation interval to 1.0 (as for each simulation step time goes by 1 second but you told the simulator to generate a vehicle every 0.3 second)

**Comment by oroojlooy** [comment]
Yes, you are right. I got a new file `flow_turn.json`, and I was looking into `flow.json`. 
But, still it generates 12 flows. How can I get 1000 flow?
